docs/flags.md

```diff
@@ -117,15 +118,17 @@ also be set:
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <compilerId>javac-with-errorprone</compilerId>
           <showWarnings>true</showWarnings>
           <compilerArgs>
-            <arg>-Xep:DeadException:WARN</arg>
-            <arg>-Xep:GuardedBy:OFF</arg>
+            <arg>-XDcompilePolicy=simple</arg>
+            <arg>-Xplugin:ErrorProne -Xep:DeadException:WARN -Xep:GuardedBy:OFF</arg>
           </compilerArgs>
         </configuration>
       </build>
     </plugins>
   </plugin>
 </project>
 ```
+
+Be aware that when running on JDK 8 the flags cannot be wrapped across multiple
+lines. JDK 9 and above do allow the flags to be separated by newlines.
```

Comment:
I'm not sure what this means, was this a JDK change or a maven change? Is it saying that e.g.

```
<arg>-Xplugin:ErrorProne
-Xep:DeadException:WARN
-Xep:GuardedBy:OFF</arg>
```

is supported?